engine/src/main/resources/templates/OCM/genericInstantEmailBodyV2.html

```diff
@@ -55,6 +55,11 @@
     <p>
         <a href="https://www.openshift.com/products/dedicated/" target="_blank">Learn more</a> about OpenShift Dedicated, and create a new cluster at any time in <a href="https://cloud.redhat.com/openshift/create" target="_blank">OpenShift Cluster Manager</a>!
     </p>
+{#case 'ocm-approved-access-template'}
+    <p>
+        Your organization has enabled "Approved Access" for ROSA clusters. Red Hat SRE cannot access any cluster in your organization unless it is approved.
+        As a reminder, the ROSA SLA can be affected if access to a cluster is not approved in a timely manner. More information can be found on this <a href="" target="_blank">here</a>.
```

Comment:
Link for the docs is still pending for this feature. I will raise another PR once I have this.